<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>771</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="InformationsClient">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Informations Client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>-1</y>
          <width>851</width>
          <height>551</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="title">
         <string>Informations Client</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>831</width>
           <height>501</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLineEdit" name="adresseDepartLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="nomLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="adresseArriveeLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Inventaire">
       <attribute name="title">
        <string>Inventaire</string>
       </attribute>
       <widget class="QTextEdit" name="inventaireTextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>851</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>841</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="SaisirInventaire">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>861</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Saisissez votre inventaire</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="AnalyseInventory">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>130</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Analyser inventaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>159</y>
          <width>851</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Elements detectes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="DevisResults">
       <attribute name="title">
        <string>Résultats et Devis</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>831</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Détails du volume</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <widget class="QWidget" name="formLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>801</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="volumeResultatLabel">
               <property name="text">
                <string>Volume total</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>50</y>
              <width>821</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>Coûts estimés</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <widget class="QWidget" name="formLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>791</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="coutMOResultatLabel">
                <property name="text">
                 <string>Coût main d'oeuvre</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="totalResultatLabel">
                <property name="text">
                 <string>Total résultat</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="coutTransportResultatLabel">
                <property name="text">
                 <string>Coût transport</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>190</y>
              <width>261</width>
              <height>111</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="genererDevisButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Générer devis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="calculerButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Calculer</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actioneeee">
   <property name="text">
    <string>eeee</string>
   </property>
  </action>
  <action name="actioneffg">
   <property name="text">
    <string>effg</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
